conf/messages.cy

```diff
@@ -399,37 +399,37 @@ property.details.cases.table.challenge=Her
 property.dvr.challenge.error=Os ydych chi’n dal i feddwl bod gennych chi’r hawl i wneud Her,   bydd angen i chi <a class="govuk-link" href="{0}"> lawrlwytho a llenwi ffurflen Her (T012)</a>
 
 assessments.table.caption=Prisiadau
-assessments.table.1=Dyddiad dod i rym
-assessments.table.2=Gwerth ardrethol
-assessments.table.3=Capasiti
-assessments.table.4=O
-assessments.table.5=Hyd
-assessments.table.6=Gweithredoedd
-assessments.table.action=Gweithred
+assessments.table.1=Prisiadau
+assessments.table.2=Dyddiad dod i rym
+assessments.table.3=Cysylltiad a r eiddo
+assessments.table.4=Gwerth ardrethol
 assessments.enddate.present=Presennol
 assessments.enddate.historic.empty=31 Mawrth 2023
 assessments.view.valuation=Gweld prisiad
 assessments.view.summary=Gweld prisiad cryno
 label.viewAssessmentRadio=Prisiad
 
 assessments.localAuthorityRef=Cyfeirnod yr awdurdod lleol:
-assessments.property=Eiddo:
+assessments.caption=Eich eiddo

```

Comment:
The 'assessments.caption' message key doesn't look like it is being used anywhere